src/lib.rs

```diff
@@ -2176,3 +2182,22 @@ impl<const N: usize> io::Write for SmallVec<u8, N> {
         Ok(())
     }
 }
+
+#[cfg(feature = "get-size")]
+impl<A: Array> GetSize for SmallVec<A>
```

Comment:
The generic types for SmallVec have changed in `v2`.  This should now be `impl<T, const N: usize>`.